x-pack/legacy/plugins/actions/server/builtin_action_types/slack.ts

```diff
@@ -42,7 +42,9 @@ export function getActionType(
 ): ActionType {
   return {
     id: '.slack',
-    name: 'slack',
+    name: i18n.translate('xpack.actions.builtin.slack.name', {
```

Comment:
```suggestion
    name: i18n.translate('xpack.actions.builtin.slackTitle', {
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/47843/commits/85578a4fc084818300dbc9fe472dab2050dce9a5.